SOURCES/privacyidea.conf

```diff
@@ -26,10 +26,13 @@ WSGIDaemonProcess privacyidea processes=1 threads=15 display-name=%{GROUP} user=
 WSGIProcessGroup privacyidea
 WSGIPassAuthorization On
 WSGIScriptAlias / /etc/privacyidea/privacyideaapp.wsgi
-SSLEngine On
-SSLProtocol All -SSLv2 -SSLv3
+
+SSLEngine on
+SSLProtocol ALL -SSLv2 -SSLv3 -TLSv1 -TLSv1.1
 SSLHonorCipherOrder On
-SSLCipherSuite EECDH+AES256:DHE+AES256:EECDH+AES:EDH+AES:-SHA1:EECDH+RC4:EDH+RC4:RC4-SHA:AES256-SHA:!aNULL:!eNULL:!EXP:!LOW:!MD5
+SSLCipherSuite          ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256
+SSLCompression          off
+SSLSessionTickets       off
```

Comment:
Please add a comment why SSL session tickets are turned off (link to https://httpd.apache.org/docs/2.4/mod/mod_ssl.html#sslsessiontickets should suffice).